### PR TITLE
Make Custom Formula easier to use

### PR DESCRIPTION
**Pull request by vtt-lair**
I've added an expander below the Custom Formula input field that has the following:

- Dice icons that when left click adds the dice to the custom formula input. Right click removes the dice
- A + and - button to add positive or negative bonuses to the custom formula input
- A list of abilities (and proficiency for D&D5E) that can be selected. Selecting said ability will add the correct modifier to the custom formula field.
- A Clear Custom Formula button that clears the custom formula field.

_Icons reused from Dice Tray with asacolips' permission_

image

image

### CODE REVIEW

**Comment by vtt-lair** [comment]
This obviously doesn't include changes for PR #107 as yet

File: src/requestor.js [hunk 1]
```diff
@@ -3,7 +3,23 @@
 class LMRTFYRequestor extends FormApplication {
     constructor(...args) {
         super(...args)
-        game.users.apps.push(this)
+        game.users.apps.push(this);
```
Comment (Varriount) [inline src/requestor.js:6]: Is this required? I haven't seen any other applications do this before.

Comment (vtt-lair) [reply src/requestor.js:6]: I didn't add that. Seems like I just added the semicolon behind the line when I was on autopilot. I can remove the semicolon if needed, but the line of code itself was there to begin with.

Comment (Varriount) [reply src/requestor.js:6]: Hm. You don't need to revert the change, though I do wonder why that is there.

**Review by Varriount — APPROVED** [review]

**Pull request merged**